test/di-order.js

```diff
@@ -15,19 +15,17 @@ var commonFalsePositives = require('./utils/commonFalsePositives');
 var eslintTester = new RuleTester();
 eslintTester.run('di-order', rule, {
     valid: [
-        'app.controller("", function($http, $q) {});',
-        'app.directive("", function($http, $q) {});',
```

Comment:
Will the 'app.*' Syntax work as well? Although there is a rule to check module getters, this syntax should not be enforced by another rule.